### PR TITLE
Feature/new reload command

### PR DESCRIPTION
**Pull request by D151l**

### CODE REVIEW

File: simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt [hunk 1]
```diff
@@ -40,39 +40,56 @@ import kotlin.concurrent.thread
...
-            val wrapperUpdater = cachedWrapper.getUpdater()
-            wrapperUpdater.setMaxSimultaneouslyStartingServices(it.getMaxSimultaneouslyStartingServices())
-            wrapperUpdater.setMaxMemory(it.getMaxMemory())
-            wrapperUpdater.update()
-        }
-        loadedWrappers.forEach { commandSender.sendProperty("manager.command.reload.wrapper-success", it.getName()) }
+    @CommandSubPath("modules", "Reload all modules")
+    fun handleReloadModules(commandSender: ICommandSender) {
+        this.reloadModules()
+    }
```
Comment (FllipEis) [inline simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt:55]: Please add a message

File: simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt [hunk 2]
```diff
@@ -40,39 +40,56 @@ import kotlin.concurrent.thread
...
 
-        //service versions
-        (CloudAPI.instance.getServiceVersionHandler() as ManagerServiceVersionHandler).reloadServiceVersions()
+    @CommandSubPath("wrappers", "Reload all wrappers")
+    fun handleReloadWrappers(commandSender: ICommandSender) {
+        this.reloadWrappers(commandSender)
+    }
```
Comment (FllipEis) [inline simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt:60]: Please add a message

File: simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt [hunk 3]
```diff
@@ -40,39 +40,56 @@ import kotlin.concurrent.thread
...
+    @CommandSubPath("jvm-arguments", "Reload jvm-arguments")
+    fun handleReloadJvmArguments(commandSender: ICommandSender) {
+        this.reloadJvmArguments()
+        commandSender.sendProperty("manager.command.reload.jvm.arguments")
+    }
+
+    @CommandSubPath("groups", "Reload all groups")
+    fun handleReloadGroups(commandSender: ICommandSender) {
+        this.reloadGroups(commandSender)
+    }
```
Comment (FllipEis) [inline simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt:77]: Please add a message

File: simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt [hunk 4]
```diff
@@ -40,39 +40,56 @@ import kotlin.concurrent.thread
...
+
+        if (module == null) {
+            commandSender.sendProperty("manager.command.reload.module.not-exists")
+            return
+        }
 
-        //groups
+        Manager.instance.cloudModuleHandler.unloadModule(module.cloudModule)
+        Manager.instance.cloudModuleHandler.loadSingleModuleFromFile(module.file)
+    }
```
Comment (FllipEis) [inline simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt:90]: Please add a message

File: simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt [hunk 5]
```diff
@@ -99,17 +121,32 @@ class ReloadCommand : ICommandHandler {
         ) { Manager.instance.cloudModuleHandler.loadAllUnloadedModules() }
     }
```
Comment (FllipEis) [inline simplecloud-base/src/main/kotlin/eu/thesimplecloud/base/manager/commands/ReloadCommand.kt:122]: Please add a message

**Review by FllipEis — CHANGES_REQUESTED** [review]

**Comment by D151l** [comment]
The missing messages are already sent elsewhere in the cloud, so I don't need to insert additional messages.

**Review by FllipEis — APPROVED** [review]

**Pull request merged**